src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -268,59 +271,57 @@ void Engine::apply_performance_hints(ov::AnyMap& config, const std::shared_ptr<o
 
     // save hints parameters to model rt_info
     ov::AnyMap hints_props;
-    const auto latency_name = std::string(CONFIG_VALUE(LATENCY)) + "_" + std::string(ov::num_streams.name());
-    const auto tput_name = std::string(CONFIG_VALUE(THROUGHPUT)) + "_" + std::string(ov::num_streams.name());
-    hints_props.insert({latency_name, latency_hints.second});
+    const auto latency_name =
+        ov::util::to_string(ov::hint::PerformanceMode::LATENCY) + "_" + std::string(ov::num_streams.name());
+    const auto tput_name =
+        ov::util::to_string(ov::hint::PerformanceMode::THROUGHPUT) + "_" + std::string(ov::num_streams.name());
+    hints_props.insert({latency_name, latency_hints});
     hints_props.insert({tput_name, std::to_string(tput_hints.second.num_streams)});
     model->set_rt_info(hints_props, "intel_cpu_hints_config");
 
     const auto perf_hint_name = getPerfHintName();
-    if (perf_hint_name == CONFIG_VALUE(LATENCY)) {
-        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = latency_hints.first;
-        config[ov::num_streams.name()] = latency_hints.second;
-    } else if (perf_hint_name == CONFIG_VALUE(THROUGHPUT)) {
+    if (perf_hint_name == ov::util::to_string(ov::hint::PerformanceMode::LATENCY)) {
+        OPENVINO_SUPPRESS_DEPRECATED_START
+        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = CONFIG_VALUE(CPU_THROUGHPUT_NUMA);
+        OPENVINO_SUPPRESS_DEPRECATED_END
+        config[ov::num_streams.name()] = latency_hints;
+    } else if (perf_hint_name == ov::util::to_string(ov::hint::PerformanceMode::THROUGHPUT)) {
+        OPENVINO_SUPPRESS_DEPRECATED_START
         config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = tput_hints.first;
+        OPENVINO_SUPPRESS_DEPRECATED_END
         config[ov::num_streams.name()] = tput_hints.first;
-        config[CONFIG_KEY_INTERNAL(BIG_CORE_STREAMS)] = std::to_string(tput_hints.second.big_core_streams);
-        config[CONFIG_KEY_INTERNAL(SMALL_CORE_STREAMS)] = std::to_string(tput_hints.second.small_core_streams);
-        config[CONFIG_KEY_INTERNAL(THREADS_PER_STREAM_BIG)] =
-            std::to_string(tput_hints.second.threads_per_stream_big);
-        config[CONFIG_KEY_INTERNAL(THREADS_PER_STREAM_SMALL)] =
+        config[ov::internal::big_core_streams.name()] = std::to_string(tput_hints.second.big_core_streams);
+        config[ov::internal::small_core_streams.name()] = std::to_string(tput_hints.second.small_core_streams);
+        config[ov::internal::threads_per_stream_big.name()] = std::to_string(tput_hints.second.threads_per_stream_big);
+        config[ov::internal::threads_per_stream_small.name()] =
             std::to_string(tput_hints.second.threads_per_stream_small);
-        config[CONFIG_KEY_INTERNAL(SMALL_CORE_OFFSET)] = std::to_string(tput_hints.second.small_core_offset);
+        config[ov::internal::small_core_offset.name()] = std::to_string(tput_hints.second.small_core_offset);
     }
     OPENVINO_SUPPRESS_DEPRECATED_END
```

Comment:
should this suppress be removed if new suppress expressions are used inside `if/else`?

Reply:
yes, good finding.